mirdata/datasets/fma_keys.py

```diff
@@ -0,0 +1,280 @@
+        url="https://zenodo.org/records/10719860/files/070-079.zip?download=1",
+        checksum="186643746fcb1f4722a28d3eb9c6b99c",
+    ),
+    "tracks-080-089": download_utils.RemoteFileMetadata(
+        filename="080-089.zip",
+        url="https://zenodo.org/records/10719860/files/080-089.zip?download=1",
+        checksum="8cf882609fc2f301621c2e9f9da03214",
+    ),
+    "tracks-090-099": download_utils.RemoteFileMetadata(
+        filename="090-099.zip",
+        url="https://zenodo.org/records/10719860/files/090-099.zip?download=1",
+        checksum="84f0f036e3778ffd97c10b591f803d06",
+    ),
+    "tracks-100-109": download_utils.RemoteFileMetadata(
+        filename="100-109.zip",
+        url="https://zenodo.org/records/10719860/files/100-109.zip?download=1",
+        checksum="4a307f019d3354064814f05d1dffa1e2",
+    ),
+    "tracks-110-124": download_utils.RemoteFileMetadata(
+        filename="110-124.zip",
+        url="https://zenodo.org/records/10719860/files/110-124.zip?download=1",
+        checksum="88d7dbcca82189ed75b7baa5aa132fc1",
+    ),
+}
+
+KEY_MAP = {
+    "C": 0,
+    "C#": 1,
+    "D": 2,
+    "D#": 3,
+    "E": 4,
+    "F": 5,
+    "F#": 6,
+    "G": 7,
+    "G#": 8,
+    "A": 9,
+    "Bb": 10,
+    "B": 11,
+}
+
+MODE_MAP = {"minor": 0, "Major": 1}
+
+
+class Track(core.Track):
+    """FMA Keys Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        spotify_uri (str): Spotify URI if available
+        key (str): path to the track's audio file
+        mode (str): path to the track's audio file
+        key_number (int): path to the track's audio file
+        mode_number (int): path to the track's audio file
+        audio (str): path to the track's audio file
+    """
+
+    def __init__(self, track_id, data_home, dataset_name, index, metadata):
+        super().__init__(track_id, data_home, dataset_name, index, metadata)
+
+        self.audio_path = self.get_path("audio")
+
+    @property
+    def spotify_uri(self):
+        return self._track_metadata.get("spotify_uri")
+
+    @property
+    def key(self):
+        return self._track_metadata.get("key")
+
+    @property
+    def mode(self):
+        return self._track_metadata.get("mode")
+
+    @property
+    def key_number(self):
+        return self._track_metadata.get("key_number")
+
+    @property
+    def mode_number(self):
+        return self._track_metadata.get("mode_number")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            metadata=self._track_metadata,
+        )
```

Comment:
You `jams_converter` is missing the `audio_path`, so it cannot compute needed attributes of the corresponding audio. See this exemple:

```
return jams_utils.jams_converter(
    audio_path=self.audio_path,
    metadata={
        "instrument": self.instrument,
        "genre": self.genre,
        "drum": self.drum,
        "train": self.train,
    },
)

That will be needed for the tests to pass!
```